session/session_test.go

```diff
@@ -3161,6 +3161,32 @@ func (s *testSessionSuite2) TestStmtHints(c *C) {
 	c.Assert(tk.Se.GetSessionVars().GetReplicaRead(), Equals, kv.ReplicaReadFollower)
 }
 
+// Test memory track in POINT_GET/BATCH_POINT_GET for issue 21653
+func (s *testSessionSuite2) TestPointGetMemoryTracking(c *C) {
```

Comment:
1. Could you directly validate the tracked memory usage?
2. Please also add tests for clustered index whose type is not an integer.

Reply:
done

Reply:
I can't directly validate the memory usage,do you have some advices to do it?

Reply:
In this case, what is the difference between integer and others, why need add tests for clustered index whose type is not an integer.